tmt-app/src/Components/MajorCareerList.jsx

```diff
@@ -1,7 +1,7 @@
 import { React, Fragment } from "react";
 import { FlexContainer } from "../Styling/CustomStyling";
 import { Box, Typography, Link } from "@mui/material";
-import { CAREERS_LIST, MAJORS_LIST } from "../constants/careerMajorList";
+import { CAREERS_LIST, MAJORS_LIST } from "../constants/constants";
```

Comment:
Ooh

![image](https://user-images.githubusercontent.com/12107969/236308638-8c493d96-e493-4f11-acdc-4274990e55d3.png)

So we could have a path of `src/utils/constants.js` might be a good place. right now we are importing from `constants/constants` and redundant